ceno_zkvm/src/instructions/riscv/ecall_insn.rs

```diff
@@ -39,9 +39,8 @@ impl EcallInstructionConfig {
 
         cb.lk_fetch(&InsnRecord::new(
             pc.expr(),
-            (EANY.codes().opcode as usize).into(),
```

Comment:
`EANY` should disappear as a kind, and be replaced with `ECALL` and `EBREAK`.

`EANY` ain't even an instruction that appears in Risc-V.  It's just an artefact of Risc0's design decisions.

(But we can do that in a subsequent PR.)

Reply:
That makes sense.